CHANGELOG.md

```diff
@@ -37,152 +37,152 @@ CHORES:
 
 ## [v0.10.0](https://github.com/vmware/terraform-provider-vcf/releases/tag/v0.10.0)
 
-> Release Date: July 9 2024
+> Release Date: 2024-07-09
 
```

Comment:
IMHO July 9 2024 is much more understandable. I'm not american and I always wonder what does 2024-07-09 mean? Is it the 7th of September of the 9th of July?